__tests__/app.test.js

```diff
@@ -34,7 +35,43 @@ describe('GET /api/topics', () => {
             response.body.topics.forEach((topic) => {
             expect(typeof topic.description).toBe('string');
             expect(typeof topic.slug).toBe('string');
+            expect.objectContaining({
+                description: 'The man, the Mitch, the legend',
+                slug: 'mitch'
+              })
         });
```

Comment:
objectContaining works well here - instead of hardcoding the values the tests will be more robust if you use this syntax - 'description: expect.any(String)'